core/src/parser.rs

```diff
@@ -593,11 +590,18 @@ fn get_field_decorators(attrs: &[syn::Attribute]) -> HashMap<SupportedLanguage,
                 language,
                 list.iter()
                     .flat_map(|nested| match nested {
-                        NestedMeta::Meta(Meta::Path(path)) => path.get_ident(),
+                        NestedMeta::Meta(Meta::Path(path)) => {
+                            Some(FieldDecorator::Word(path.get_ident()?.to_string()))
+                        }
+                        NestedMeta::Meta(Meta::NameValue(name_value)) => {
+                            Some(FieldDecorator::NameValue(
+                                name_value.path.get_ident()?.to_string(),
+                                literal_as_string(name_value.lit.clone())?,
+                            ))
+                        }
                         _ => None,
```

Comment:
It's likely that this branch should return an error instead of silently failing, since it indicates a malformed `#[typeshare]` attribute. This can be left to a future date, though.